rpm-builder/rippled.spec

```diff
@@ -90,6 +98,14 @@ chmod 755 /var/lib/rippled/
 %dir /var/log/rippled/
 %dir /var/lib/rippled/
 
+%files devel
+%{_prefix}/include
+%{_prefix}/lib/*.a
+%{_prefix}/lib/cmake/ripple
+
 %changelog
 * Thu Jun 02 2016 Brandon Wilson <bwilson@ripple.com>
 - Install validators.txt
+
+* Wed Aug 1 2018 Mike Ellery <mellery451@gmail.com>
+- add devel package for signing library
```

Comment:
```
error: %changelog not in descending chronological order
```
I think you can use [rpmdev-bumpspec](https://linux.die.net/man/1/rpmdev-bumpspec) to update the changelog.